gr-uhd/python/uhd/bindings/uhd_types_python.cc

```diff
@@ -141,8 +141,8 @@ void bind_uhd_types(py::module& m)
 
     // ::uhd::tune_request_t
     using tune_request_t = ::uhd::tune_request_t;
-    py::class_<tune_request_t>(m, "tune_request_t")
-        .def(py::init<double, double>())
+    py::class_<tune_request_t> tune_request(m, "tune_request_t");
+    tune_request.def(py::init<double, double>())
```

Comment:
What is this change for?

Reply:
usually the class definition gets broken up that way to shove an enum in with the class, but nothing is being added here.  the change is not harmful, but also not necessary

Reply:
I needed to reference the tune_request python class to put the policy_t enum within it. I was trying to be consistent with the uhd hierarchy.

I now see that other uhd enums are just defined at the module level. Would this be preferred over nesting it within tune_request?

Reply:
Yes, what you did makes sense - that is indeed the proper way to get the enum nested under the class name in the bindings.